src/utils/testResources.ts

```diff
@@ -0,0 +1,61 @@
+import { Pokemon, PokemonData, PokemonDetails } from 'types';
```

Comment:
probably it is constants, so let's move it to appConstants

Reply:
Actually, it shouldn't be in constants and utils as well. I decided to follow the approach where people store stuff like that in __mocks__ folder